src/components/DonutGraph.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled from "styled-components";
+
+import { CircularProgressbar } from "react-circular-progressbar";
+import "react-circular-progressbar/dist/styles.css";
+
+const DonutGraph = React.memo(({ percent }) => {
+  // todos 항목이 없는 빈 배열일 경우 0%로 초기화
+  if (isNaN(percent)) {
+    percent = 0;
+  }
```

Comment:
props를 자식컴포넌트에서 직접 수정하는 것보다 이렇게 변수를 만들어서 하는 것도 좋은 방법일 것 같아요

```suggestion
const displayPercent = isNaN(percent) ? 0 : percent;

```

Reply:
별도의 의미를 가진 변수와 함께 한 줄로 훨씬 깔끔하게 쓸 수 있군요!